packages/mdc-elevation/README.md

```diff
@@ -44,6 +44,18 @@ npm install --save @material/elevation
 
 ## Usage
 
+### CSS Classes
+
+Some components have a set elevation. For example, a raised MDC Button has elevation 2.
+
+However, if you want to set the elevation of an element, which is not an Material Design component, then you
```

Comment:
"~However~ If you want... ...not _a_ Material Design component, ~then~ you can apply..."